Detectors/ITSMFT/common/simulation/include/ITSMFTSimulation/DigiParams.h

```diff
@@ -96,7 +96,7 @@ class DigiParams
   const SignalShape& getSignalShape() const { return mSignalShape; }
   SignalShape& getSignalShape() { return (SignalShape&)mSignalShape; }
 
-  void print() const;
+  virtual void print() const;
```

Comment:
Are going to derive from this class and redefine the print method? If so, then the class cannot use ClassDefNV, should be changed to ClassDef.

Reply:
No, just use ClassDef, the ClassDefNV is reserved for classes w/o vtable,

Reply:
Thanks btw! I increased the version number since it the difference between ClassDefNV and ClassDef is the definition of the `Streamer(TBuffer&)` function which either is virtual or not.